frontend/www/js/omegaup/components/contest/Edit.test.ts

```diff
@@ -85,11 +88,12 @@ describe('Edit.vue', () => {
 
   it('Should handle a virtual contest from an original private contest', () => {
     propsData.details.rerun_id = 2;
+    propsData.initialTab = '';
     propsData.originalContestAdmissionMode = 'private';
     const wrapper = shallowMount(contest_Edit, {
       propsData,
     });
 
-    expect(wrapper.vm.showTab).toBe('links');
+    expect(wrapper.vm.showTab).toBe('problems');
```

Comment:
According the previous comment, this tab should be `links`:
```suggestion
    expect(wrapper.vm.showTab).toBe('links');
```